programs/bpf_loader/src/syscalls.rs

```diff
@@ -934,901 +1150,1892 @@ impl<'a> SyscallObject<BpfError> for SyscallSha256<'a> {
     }
 }
 
-fn get_sysvar<T: std::fmt::Debug + Sysvar + SysvarId>(
```

Comment:
Not sure why github is breaking the diff up like this but it's making it very hard to read